app/cypress/e2e/api-request.cy.ts

```diff
@@ -207,7 +207,7 @@ describe("Api Validations", () => {
       "organizationName",
       "otherIdentifier",
       "region",
-      // "tmdlEndpoint", 404 error
+      "tmdlEndpoint",
```

Comment:
```suggestion
```

This can be removed as it will be removed from the database. I'll go ahead and push a commit for this one.